src/jit/morph.cpp

```diff
@@ -2564,6 +2602,7 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
     regMaskTP       argSkippedRegMask    = RBM_NONE;
 
 #if defined(_TARGET_ARM_) || defined(_TARGET_AMD64_)
+    // Note this in ONLY used by _TARGET_ARM_
     regMaskTP       fltArgSkippedRegMask = RBM_NONE;
```

Comment:
This is an observation that I made about the 'fltArgSkippedRegMask' variable.
I added this comment so that I would be motivate to correct this in the future.

As this wasn't directly relevant to the multi-reg return work, I didn't change the ifdefs for this with this checkin.